### PR TITLE
docs: add section about focus outlines

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: docs/GETTING_STARTED.md [hunk 1]
```diff
@@ -305,6 +305,28 @@ All our tokens and components use `rem` as size unit.
 We strongly recommend using the `rem` unit to guarantee consistent scaling
 when font size changes in browsers.
 
+### Focus outline
+
+Focus outlines should only be displayed when users are navigation by keyboard.
```
Comment (TomMenga) [inline docs/GETTING_STARTED.md:310]:
```suggestion
Focus outlines should only be displayed when users are navigating by keyboard.

Or

Focus outlines should only be displayed when users navigate by keyboard.
```

**Review by TomMenga — CHANGES_REQUESTED** [review]
Just a typo

File: docs/GETTING_STARTED.md [hunk 2]
```diff
@@ -305,6 +305,32 @@ All our tokens and components use `rem` as size unit.
 We strongly recommend using the `rem` unit to guarantee consistent scaling
 when font size changes in browsers.
 
+### Focus outline
+
+Focus outlines should only be displayed when users are navigating by keyboard.
+
+Or
+
+Focus outlines should only be displayed when users navigate by keyboard.
```
Comment (TomMenga) [inline docs/GETTING_STARTED.md:314]:
```suggestion
Focus outlines should only be displayed when users are navigating by keyboard.
```

**Review by TomMenga — CHANGES_REQUESTED** [review]
my bad, the suggestion was a choice between 2 options

**Comment by jeripeierSBB** [comment]
> my bad, the suggestion was a choice between 2 options

Oops, I was too fast :D

**Review by TomMenga — APPROVED** [review]
LGTM 👍

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**